apps/web/src/studio/components/workflows/preferences/WorkflowDetailFormContextProvider.tsx

```diff
@@ -26,14 +26,6 @@ export const WorkflowDetailFormContextProvider: FC<PropsWithChildren<IWorkflowDe
   const formValues = useForm<WorkflowDetailFormContext>({
     mode: 'onChange',
     defaultValues: DEFAULT_FORM_VALUES,
-    values: {
-      general: {
-        workflowId: '',
-        name: '',
-        description: undefined,
-      },
-      preferences: DEFAULT_WORKFLOW_PREFERENCES,
-    },
```

Comment:
There is no need to set the values here, we set from the default values and then set the values from the workflow